modules/sublimeBidAdapter.js

```diff
@@ -134,22 +137,40 @@ export const spec = {
      * Unpack the response from the server into a list of bids.
      *
      * @param {*} serverResponse A successful response from the server.
+     * @param {*} bidRequest An object with bid request informations
      * @return {Bid[]} An array of bids which were nested inside the server.
      */
-  interpretResponse: (serverResponse) => {
+  interpretResponse: (serverResponse, bidRequest) => {
     // debug pixel interpret response
     sendAntennaPixel('dintres');
 
     const bidResponses = [];
     const response = serverResponse.body;
 
     if (response) {
+      // Setting our returned sizes object to default values
+      let returnedSizes = {
+        width: 1800,
+        height: 1000
+      };
+
+      // Getting banner sizes, if not defined we set bannerSize to false
+      const bannerSize = bidRequest ? bidRequest.data.meta.mediaTypes.banner.sizes[0] || false : false;
```

Comment:
We need tests here, the case when banner is null or when bannerSize = false.

Reply:
This should not be the case anymore since bidRequest now use data.w and data.h which are always set from buildRequest()